yoda_dbt2looker/generator.py

```diff
@@ -398,13 +419,70 @@ def lookml_non_aggregative_measure(measure: models.Dbt2LookerExploreMeasure):
 
 
 def lookml_calculated_dimension(dimension: models.Dbt2LookerExploreDimension):
-    return {
+    d = {
```

Comment:
can the hidden key be always added with None in case if dimension.hidden evaluated to false ? ( the same for the code in lookml_parameter_exposure and lookml_filter_exposure)  -  If not , consider providing changing the "d" variable to a temp or something which will make more sense